src/lib/helpers.ts

```diff
@@ -130,7 +130,7 @@ export async function getBranchPrompt(branch, defaultName?) {
 export async function getTagPrompt(tag, msg, nextRelease?) {
   if (!tag) {
     const latestTagCommit = exec('git rev-list --tags --max-count=1', { log: false });
-    const latestTag = await exec(`git describe --tags ${latestTagCommit}`, { log: false });
+    const latestTag = await exec(`git describe --tags --match "v[0-9]*" ${latestTagCommit}`, { log: false });
```

Comment:
This won't work with java repos that don't add a `v` in their tag.

Reply:
Maybe tweak the regex a bit? Something like `(v|[0-9])*`?

Reply:
Now that I am seeing it again it is the above command that would need to change 
`const latestTagCommit = exec('git rev-list --tags --max-count=1', { log: false });` It takes the last tag

Reply:
I think it returns a commit hash. The second command uses that hash and finds the most recent tag that is reachable from a commit.